Extension/src/LanguageServer/Providers/codeActionProvider.ts

```diff
@@ -223,7 +247,8 @@ export class CodeActionProvider implements vscode.CodeActionProvider {
                     arguments: command.arguments
                 },
                 edit: wsEdit,
-                kind: codeActionKind
+                kind: codeActionKind,
+                disabled: disabledReason ? { reason: disabledReason } : undefined
```

Comment:
This ternary operator shouldn't be needed.  `disabledReason` is `string | undefined` and you should be able to assign it directly.

Reply:
I thought it was needed to create an object with a `reason: string` property and not a `reason: string | undefined` property, but I can double check.

Reply:
You're right. I misread the code.